.github/workflows/ci.yaml

```diff
@@ -165,10 +167,24 @@ jobs:
         run: just slt 'sqllogictests/*'
 
   integration-tests:
-    if: github.repository_owner == 'glaredb'
     name: Integration Tests
     runs-on: ubuntu-latest-8-cores
     needs: ["build"]
+
+    # We skip this action for all other users since this would eventually fail
+    # for external contributors. If you are a member of GlareDB organization,
+    # add yourself to this list to run the tests.
+    #
+    # See: https://github.com/orgs/community/discussions/26622#discussioncomment-3252561
+    if: |
+      contains('[
+        "scsmithr",
+        "greyscaled",
+        "tychoish",
+        "universalmind303",
+        "vrongmeal",
+      ]', github.actor)
```

Comment:
This doesn't feel ideal. Are we able to do something like `if: ${{ !github.event.pull_request.head.repo.fork }}`?

Reply:
Yeah, updated this!